Fabric.Authorization.API/scripts/Install-Authorization-Windows.ps1

```diff
@@ -758,4 +757,25 @@ Write-Host "There are $($edwAdminUsers.Count) users with this role"
 Write-Host ""	
 Add-ListOfUsersToDosAdminRole -edwAdminUsers $edwAdminUsers -connString $authorizationDbConnStr -authorizationServiceUrl "$authorizationServiceUrl/v1" -accessToken $accessToken
 
+#Register Fabric.Authorization.AccessControl client
+$body = @'
+{
+    "clientId":"fabric-access-control", 
+    "clientName":"Fabric Authorization Access Control Client", 
+    "requireConsent":"false", 
+    "allowedGrantTypes": ["implicit"], 
+    "allowedScopes": ["openid", "profile", "fabric.profile", "fabric/authorization.read", "fabric/authorization.write", "fabric/idprovider.searchusers", "fabric/authorization.dos.write"],
+	"allowedCorsOrigins": ["$authorizationServiceUrl/v1"],
+	"redirectUris": ["$authorizationServiceUrl/v1/oidc-callback.html", "$authorizationServiceUrl/v1/silent.html"],
```

Comment:
Do we need the v1's here? I don't think users should need to put in a /v1 to get to the UI, they should just be able to enter something like https://host.domain.local/Authorization. The UI should know if its using a v1 or v2 endpoints of the API.